File: python/data-structures/stack/Stack.py

class Stack:
    def __init__(self):
        self.items = []

    def push(self, item): 
        self.items.append(item)

    def pop(self):
        return self.items.pop() if not len(self.items) == 0 else None
    
    def peek(self):
        return self.items[-1] if not len(self.items) == 0 else None

File: java/data-structures/stack/Stack.java
package stack;

public class Stack {
    int items[] = new int[5];
    int top = 0;
    
    public void push(int item) {
        items[top] = item;
        top++;
    }

    public int pop(){
        return items[top--];
    }

    public int peek(){
        return items[top];
    }
}
